### ISSUE TITLE
Normalize problem with prices attribute of SendInvoiceMethod

### ISSUE THREAD

**Issue by oRastor** [tg-bot-api/bot-api-base#33]
I tried to send an invoice, but got the notice:
PHP Notice:  Array to string conversion in .../src/ApiClient.php on line 106

And receive unsuccessful response from telegram.

Example
```
$invoice = SendInvoiceMethod::create(
    $telegramAccount->getTelegramId(),
    'title',
    'description',
    'payload',
    'token',
    'startParameter',
    'USD',
    [LabeledPriceType::create('label', 234)]
);

$bot->sendInvoice($invoice);
```

When ApiClient try to createStreamBody it cast value (array) as string:
```
        foreach ($request->getData() as $name => $value) {
            // todo [GreenPlugin] fix type cast and replace it to normalizer
            $stream .= $this->createDataStream($boundary, $name, (string) $value);
        }
```

**Comment by greenplugin** [comment]
Hi, Thanks for this issue. I'll fix it today.

**Comment by greenplugin** [comment]
Done. This bug fixed in [1.5.1](https://github.com/tg-bot-api/bot-api-base/releases/tag/1.5.1). Please update package

**Comment by oRastor** [comment]
Thank you for the quick response, it works!
